### DISCUSSION TITLE
Increasing contributors base aka increasing user base aka just make it loud about the project

### DISCUSSION THREAD

**Discussion by derberg** [asyncapi/modelina#506] [General]
As per the discussion we had here https://www.youtube.com/watch?v=48SDAJiezlQ

- enable other inputs supported by Modelina in current playground on asyncapi.com
- we need some good article about Modelina without AsyncAPI as main input, but one of many and probably not hosted on asyncapi.com but rather dev.to (and use their promotion channels)
- we should submit it for channels like https://twitter.com/JavaScriptDaily that promote different libraries
- apply for a JS-like conference
- go to Reddit and HackerNews

Prerequisites if short term fixable:
- https://github.com/asyncapi/modelina/discussions/500
- https://github.com/asyncapi/modelina/security/advisories/GHSA-4jg2-84c2-pj95

--- 
The stuff we did not talk about during the stream is https://github.com/quicktype/quicktype.
Quicktype lacks proper maintenance and basically, no responsiveness of maintainers drove us towards working on Modelina. They are popular though. Would be good to look at them closely. Look at current PRs that are hanging there for ages, maybe contributors would like to join Modelina? Google for articles, and contact authors writing about Quicktype to have a look at Modelina. 

Basically, the popularity of Quicktype shows such a tool is very much needed and the community needs it, we just need to target it directly, especially that we are not really a competition, as you cannot compete with something that authors don't care much about.
